app/src/main/java/de/westnordost/streetcomplete/quests/barrier_opening/AddGateWidthForm.kt

```diff
@@ -0,0 +1,94 @@
+package de.westnordost.streetcomplete.quests.barrier_opening
+
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AlertDialog
+import androidx.core.view.isGone
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.databinding.QuestLengthBinding
+import de.westnordost.streetcomplete.osm.Length
+import de.westnordost.streetcomplete.osm.hasDubiousRoadWidth
+import de.westnordost.streetcomplete.quests.AbstractArMeasureQuestForm
+import de.westnordost.streetcomplete.quests.width.WidthAnswer
+import de.westnordost.streetcomplete.screens.measure.ArSupportChecker
+import de.westnordost.streetcomplete.view.controller.LengthInputViewController
+import org.koin.android.ext.android.inject
+
+class AddGateWidthForm : AbstractArMeasureQuestForm<WidthAnswer>() {
```

Comment:
Can you explain why `AddWidthForm` can not be used here?

Reply:
AddWidthForm amended and used